plugins/ruby-foreman-docker/debian/gem.list

```diff
@@ -1,5 +1,5 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_docker-2.0.1.gem"
-GEMS="$GEMS https://rubygems.org/downloads/docker-api-1.17.0.gem"
+GEMS="https://rubygems.org/downloads/foreman_docker-2.1.0.gem"
+GEMS="$GEMS https://rubygems.org/downloads/docker-api-1.18.0.gem"
```

Comment:
Probably worth updating this to the latest, matching what you're doing with the RPMs.